ci/git-clean-reset.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+# Reset the repository pointed by $1 to the revision given by $2
+
+cd "$1" || {
```

Comment:
Please also verify the existence and validity of $2.
I think `git reflog` may have some validation abilities for $2.

Reply:
I think adding defaults is a good idea - we can assume that easymesh_cert is in the same level as prplmesh, and the default revision can just be the master branch.
```suggestion
path=${1-../../easymesh_cert}
revision=${2-master}
cd "$1" || {
```

Reply:
@RanRegev  
If $2 doesn't exist, git will report the reference doesn't exist, and the script will exit, so I don't think any additional checks are necessary.

@tomereli 
This script is about resetting any git repository, it has nothing specific to the easymesh_cert repo. The default value you mention is already in `run_easymesh-cert.sh`